src/control/cmd/dmg/main.go

```diff
@@ -121,8 +121,9 @@ func appSetup(log logging.Logger, broadcast bool, opts *cliOptions, conns client
 	// broadcast app requests to host list by default
 	addresses := config.HostList
 	if !broadcast {
-		// send app requests to first access point only
-		addresses = []string{config.AccessPoints[0]}
+		// send app requests to first element in hostlist (assumed to be
```

Comment:
I think the better/future-proof way to handle this is to remove the whole broadcastCmd/connectedCmd thing and instead give the config.HostList array to the Connect implementation, which takes care of figuring out a) Is this a command which needs to be sent to the current service leader or b) broadcast to the supplied host list.

If it's a command which goes to the MS Leader (pool operations), then the current selection algorithm just selects the first host in the list.